platform/representation/src/gestures/DropGesture.js

```diff
@@ -165,16 +165,18 @@ define(
                     if (shouldCreateVirtualPanel(domainObject, selectedObject)){
```

Comment:
I would not be opposed to this, especially given that it doesn't work in VISTA and actually just complicates things there. It was just a prototype after all. 

I will leave it enabled in a branch if we ever want to demo it in WARP.